gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjureGeneratorTask.java

```diff
@@ -60,6 +64,7 @@ final void setExecutablePath(Supplier<File> executablePath) {
     }
 
     @InputFile
+    @PathSensitive(PathSensitivity.NONE)
```

Comment:
Why did we make this path insensitive? Doesn't the path change when the executable changes?

Reply:
We'd want to use one of the others if the path name could be the _only_ thing that changes.  I don't think that's every the case though, as the contents of the file is good enough to determine whether we should re-run or not